types.go

```diff
@@ -446,3 +551,37 @@ func (d *StringOrUint64) UnmarshalJSON(b []byte) error {
 	*d = StringOrUint64(parsed)
 	return nil
 }
+
+type NodeNetworks []*NodeNetwork
+type NodeNetwork struct {
+	client  *Client `json:"-"`
+	Node    string  `json:"-"`
+	NodeApi *Node   `json:"-"`
+
+	Iface    string `json:"iface,omitempty"`
+	BondMode string `json:"bond_mode,omitempty"`
+
+	Autostart int `json:"autostart,omitempty"`
+
+	Cidr            string `json:"cidr,omitempty"`
```

Comment:
same comment about all caps for acronyms, ok to do `CIDR`, `CIDR6`?

Reply:
updated,thanks